tests/components/ct-angular/tests/events.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/experimental-ct-angular';
+import { ButtonComponent } from '@/components/button.component';
+import { OutputComponent } from '@/components/output.component';
+
+test('emit an submit event when the button is clicked', async ({ mount }) => {
+  const messages: string[] = [];
+  const component = await mount(ButtonComponent, {
+    props: {
+      title: 'Submit',
+    },
+    on: {
+      submit: (data: string) => messages.push(data),
+    },
+  });
+  await component.click();
```

Comment:
This line does not click the button.

Suggestion:
```ts
component.locator('css=button').click()
```

Reply:
The current test is correct _(and passing)_.